packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,181 @@
+# Alert
+
+Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a **prominent way**.
+
+This document outlines the API of `Alert` component.
+
+<img src="./alerts-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6824%3A0)
+
+> **Note**
+>
+> The following needs work:
+>
+> 1. color on banking theme in high contrast
+> 2. a11y contrast ratio for notice alerts in high contrast
+> 3. the borderless variant mentioned in full bleed layout shouldn't have any borders and border radii
+
+## API
+
+Annotated component with props:
+
+![Alert breakdown](./alert-breakdown.png)
+
+Sample usage:
+
+```jsx
+import { Alert, Link } from '@razorpay/blade';
+
+<Alert title="International Payments Only">
+  <Alert.Description>
```

Comment:
Also I'm not a great fan of the dot notation components. 

The problem with this approach is: 

1. It now becomes a backpack component. Say for example users never use Alert.SecondaryAction they will still carry the bundle size with them. They are not treeshakable. 
2. Discoverability issues are there with this approach, since now users have no idea that pressing "." dot after writing alert will have other components inside of it. 

I'll rather just do this: 

```tsx
<Alert>
  <AlertDescription>hello<AlertDescription>
  <AlertPrimaryAction>click</AlertPrimaryAction>
</Alert>
```


But I do still see why it can be useful, tho we first have to discuss wether we want this pattern in all components or not. If lets say we use this pattern only for Alert and not for other components end users will just get confused on which standard practices we follow.